internal/cmd/cmd.go

```diff
@@ -44,6 +44,7 @@ func Do(args []string, stdin io.Reader, stdout io.Writer, stderr io.Writer) int
 	rootCmd.SetIn(stdin)
 	rootCmd.SetOut(stdout)
 	rootCmd.SetErr(stderr)
+	rootCmd.SilenceErrors = true
```

Comment:
When we moved to `RunE`, we accidentally added some additional error output. Turning that off for now.